edi_oca/tests/test_exchange_type_encoding.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2024 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+import base64
+
+import chardet
```

Comment:
Do we really need this test dep?
Can't we have some pre-defined txt w/ specific encoding?